skymp5-server/cpp/server_guest_lib/script_classes/PapyrusForm.cpp

```diff
@@ -133,6 +134,30 @@ VarValue PapyrusForm::GetName_(VarValue self, const std::vector<VarValue>&)
   return VarValue("");
 }
 
+VarValue PapyrusForm::GetWeight(VarValue self,
+                                const std::vector<VarValue>& arguments)
+{
+  const auto* form = GetFormPtr<MpForm>(self);
```

Comment:
Wiki doesn't say it works on object references or not https://ck.uesp.net/wiki/GetWeight_-_Form

And I don't know. let's assume it works so we keep this impl.

**But we have to handle the case when `GetWeight` is called on base objects like WEAP, AMMO, etc**

For that you'll probably want extracting EspmGameObject